src/js/binaryen.js-post.js

```diff
@@ -42,11 +52,12 @@ function initializeConstants() {
     ['unreachable', 'Unreachable'],
     ['auto', 'Auto']
   ].forEach(entry => {
-    Module[entry[0]] = Module['_BinaryenType' + entry[1]]();
+    const value = MODULE['_BinaryenType' + entry[1]]();
+    Types[entry[0]] = value;
+    Types[value] = entry[0];
```

Comment:
Unrelated: Adds a reverse lookup to the enums, as TypeScript typically does and expects when typed as `enum` in TS definitions.